### ISSUE TITLE
Lists aren't parsed properly

### ISSUE THREAD

**Issue by RiedleroD** [wereturtle/ghostwriter#586]
**Platform:**
 - Arch Linux 5.11.15.arch1-2
 - Wayland/Sway

**ghostwriter installation:**
 - Ghostwriter v1.8.1
 - official arch package from the community repository

**To Reproduce**

1. open ghostwriter
2. write the following:
  ```
  - first item
  - second item  
    some explainer text
  - third item
  ```

**Expected behavior**
All three list items should be recognized as such.
In github, this parses as:
- first item
- second item
  some explainer text
- third item

**Actual behaviour**
After a line after a breaking newline (shift+enter) in which no item marker is present, no further items will be recognized. In other words, the third item wouldn't be recognized as part of a list.
The preview does however show the item as a list, just like github.
The effects are that a newline (enter) doesn't automaticall place a new marker in the next line, and the markers that are manually added are slightly lighter than they're supposed to be.

**Comment by RiedleroD** [comment]
explanation please?

**Comment by wereturtle** [comment]
Sorry, I knew I forgot something!  The issue is fixed in version 2.0, which now uses cmark-gfm to assist in syntax highlighting.
